parserc.go

```diff
@@ -890,6 +893,14 @@ func yaml_parser_parse_block_mapping_value(parser *yaml_parser_t, event *yaml_ev
 	if token.typ == yaml_VALUE_TOKEN {
 		mark := token.end_mark
 		skip_token(parser)
+		// Move foot comment to head comment
```

Comment:
I can see that's being done below, but I cannot see why this is being done. This comment is a nice place to describe that, as this is not an obvious operation.